plugin/secrets_access_token.go

```diff
@@ -58,19 +57,13 @@ func (b *backend) secretAccessTokenResponse(ctx context.Context, s logical.Stora
 		return nil, errwrap.Wrapf("could not create IAM Admin client: {{err}}", err)
 	}
 
-	// Verify account still exists
-	_, err = rs.getServiceAccount(iamC)
-	if err != nil {
-		return logical.ErrorResponse(fmt.Sprintf("could not get role set service account: %v", err)), nil
-	}
-
 	if rs.TokenGen == nil || rs.TokenGen.KeyName == "" {
 		return logical.ErrorResponse(fmt.Sprintf("invalid role set has no service account key, must be updated (path roleset/%s/rotate-key) before generating new secrets", rs.Name)), nil
 	}
 
 	token, err := rs.TokenGen.getAccessToken(ctx, iamC)
 	if err != nil {
-		return logical.ErrorResponse(fmt.Sprintf("could not generate token: %v", err)), nil
+		return logical.ErrorResponse(fmt.Sprintf("unable to generate token - make sure your roleset service account and key are still valid: %v", err)), nil
```

Comment:
ErrorResponse now has format string support.

```suggestion
		return logical.ErrorResponse("unable to generate token - make sure your roleset service account and key are still valid: %v", err), nil
```